packages/neon/neon/lib/src/widgets/error.dart

```diff
@@ -11,6 +11,7 @@ import 'package:neon/src/utils/exceptions.dart';
 import 'package:neon/src/utils/provider.dart';
 import 'package:nextcloud/nextcloud.dart';
 
+/// Displays any [error] for the user.
 class NeonError extends StatelessWidget {
   const NeonError(
```

Comment:
Missing documentation for a public member.
Try adding documentation for the member.(dartpublic_member_api_docs)[https://dart.dev/tools/linter-rules/public_member_api_docs].

~ Linter